src/sentry/constants.py

```diff
@@ -455,14 +453,27 @@ def as_choices(cls):
         )
 
     @classmethod
-    def as_str(cls, status):
-        if status == cls.DISCOVER_V2:
+    def as_str_choices(cls):
```

Comment:
if you aren't using this anywhere except the serializer we can probs get rid of it and just use a constant in that file